data_filter_02.py

```diff
@@ -18,8 +18,10 @@
 	b = {}
 	b["title"] = d["title"]
 	b["num_comments"] = d["num_comments"]
-	d["id"] = d["id"]
-	d["score"] = d["score"]
+	b["id"] = d["id"]
```

Comment:
The original code
```
d["id"] = d["id"]
```
obviously does nothing, as `variable1 = variable1` is doing nothing.

Probably you meant to assign the values from the`d` dictionary to the `b` dictionary.